src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.NegativeValueException;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.strategy.FruitService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private StorageDao storage;
+    private Transaction transaction;
+    private FruitService fruitService;
+
+    @BeforeEach
+    public void setUp() {
+        storage = new StorageDaoImpl();
+        fruitService = new FruitServiceImpl(storage);
```

Comment:
why are you using your fruitService here? This test class should test BalanceOperationHandler class directly. Please fix similar issue in all tests